docs/content/config/security/mail_crypt.md

```diff
@@ -64,3 +70,71 @@ Official Dovecot documentation: https://doc.dovecot.org/configuration_manual/mai
     ```
 
 This should be the minimum required for encryption of the mail while in storage.
+
+
+## Encrypted User Keys
+
+1. Create `10-custom.conf` and populate it with the following:
+
+    ```
+    mail_attribute_dict = file:%h/Maildir/dovecot-attributes
+    mail_plugins = $mail_plugins mail_crypt
+    mail_debug= yes
```

Comment:
I don't think you need this. Most likely this prints all emails that you wanted to encrypt to syslog ;_

Reply:
Yep! Forgot to remove it :)